test/model.index.spec.js

```diff
@@ -172,6 +172,7 @@ describe('Model index function tests', function () {
     });
 
     function checkIndexRes(err, indexRes) {
+      console.dir(err);
```

Comment:
Don't use `console.dir`. It may not be printing the object in the state it's in when you invoke `console.dir`. 

https://developer.mozilla.org/en-US/docs/Web/API/Console/dir